src/core/lib/layer-manager.js

```diff
@@ -259,9 +259,11 @@ export default class LayerManager {
   }
 
   setViewState(viewState) {
-    const viewStateChanged = deepEqual(viewState, this.viewState);
-    this.viewState = viewState;
-    this.viewsChanged = true || viewStateChanged;
+    if (viewState) {
```

Comment:
Do we really want to support `null` viewState? Shouldn't we at least warn?

Reply:
want to avoid crash for this case, added the warning.